coalib/bearlib/languages/documentation/DocumentationComment.py

```diff
@@ -193,16 +201,16 @@ def from_metadata(cls, doccomment, docstyle_definition,
         ...     import DocumentationComment
         >>> from coalib.bearlib.languages.documentation.DocstyleDefinition \
         ...     import DocstyleDefinition
-        >>> from coalib.results.TextRange import TextRange
+        >>> from coalib.results.TextPosition import TextPosition
         >>> Description = DocumentationComment.Description
         >>> Parameter = DocumentationComment.Parameter
         >>> python_default = DocstyleDefinition.load("python3", "default")
         >>> parsed_doc = [Description(desc='\nDescription\n'),
         ...               Parameter(name='age', desc=' Age\n')]
         >>> str(DocumentationComment.from_metadata(
         ...         parsed_doc, python_default,
-        ...         python_default.markers[0], 4,
-        ...         TextRange.from_values(0, 0, 0, 0)))
+        ...         python_default.markers[0], "    ",
```

Comment:
Amended. Separate commit. hmm this caused that `pydocbear` test failure. separate commit it is!